src/transaction/combine.ts

```diff
@@ -5,6 +5,28 @@ import BigNumber from 'bignumber.js'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import { JsonObject } from 'ripple-binary-codec/dist/types/serialized-type'
+
+//The transactions should all be equal EXCEPT for the Signers field
```

Comment:
Can you add these types of quick explaining comments to the other methods in this file as well?

Reply:
should have have a space after `//`

Reply:
also, since you have a function that is validating this condition you can rewrite this comment to be more clear in terms of what the function is doing, as opposed to the abstract requirement. See my suggested function below.